<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HashCheckDialog</class>
 <widget class="QDialog" name="HashCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>125</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Hash Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>File   :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileEdit">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="open">
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Open file or drop here</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calculateButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Go</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="hashSelector">
       <property name="currentText">
        <string notr="true">SHA-3</string>
       </property>
       <item>
        <property name="text">
         <string notr="true">SHA-3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SHA-1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SHA-224</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SHA-256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SHA-384</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SHA-512</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SHA-512-256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Skein-512</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Keccak-1600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Whirlpool</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Blake2b</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SHAKE-128(128)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SHAKE-256(256)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">GOST-34.11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">SM3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Streebog-256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Streebog-512</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">RIPEMD-160</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Adler32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">MD4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">MD5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">CRC24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">CRC32</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="format">
        <string notr="true">%p%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="checksumEdit">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
